src/interfaces/store-interfaces/state-interfaces/tombstone-interface.ts

```diff
@@ -12,6 +12,7 @@ export interface TombstoneIF {
   userLastName: string // from auth user info
   userKeycloakGuid: string // from auth user info
   userAddress: AddressIF // from auth org info
-  accountFolioNumber: string // from auth user info
+  folioNumber: string // from auth user info or from user
   transactionalFolioNumber: string
+  transactionalFolioNumberValid: boolean
```

Comment:
This is a single flag so I put it here, but if we end with many flags and we want to centralize them, then we should create a new object to put them all into.